python/contactflowhelper/README.md

```diff
@@ -0,0 +1,251 @@
+# Contact Flow Helper
+A single AWS Lambda function which provides a set of basic math and text tools to help manipulate data in contact flows. Included operations are:
+* freeform_math = performs basic evaluation of a mathematical expression. For example: 2+2
+* increment_1 = increments a provided number by 1
+* increment_n = increments a provided number by a provided amount
+* random_choice = returns a random selection from a provided list of options
+* replace_text = replaces defined string within a text string with a provided string
+* split_text = splits a defined string into two strings at a defined point
+* strip_text = strips text from the start/end of a defined string
+* lower_text = returns the lowercase version of a string
+* upper_text = returns the uppercase version of a string
+
+__NOTE:__ No special IAM roles are required for this function.
```

Comment:
Can you add a note about the max payload? For example, if you use split_text, you may send in a small enough payload, but the lambda result could be too large and fail on the return. It would be nice if the lambda function checked that before sending a response

Reply:
Isn't the upper limit here 32K? We document that in the guide already, why add it here as well?

Reply:
the guide being documentation? i just figure it could be a nice call out to help people troubleshoot seemingly random failures faster, but isnt required